src/resources/views/web/products/suggest-js.blade.php

```diff
@@ -0,0 +1,146 @@
+<script>tinymce.init({selector: '#content'});</script>
+<script type="text/javascript">
+    tinymce.init({
+        selector: "textarea",
+        plugins: ["advlist autolink lists link image charmap print preview anchor",
+            "searchreplace visualblocks code fullscreen", "insertdatetime media table contextmenu paste"],
+        toolbar: "insertfile undo redo | styleselect | bold italic | alignleft aligncenter alignright alignjustify | " +
+            "bullist numlist outdent indent | link image"
+    });
+
+    let jcAlert = $.alert({
+        title: "{{__('custom.message')}}",
+        lazyOpen: true,
+    });
+
+    $(document).ready(function () {
+        /* Init Category */
+        let category_food = $("#categoryFood");
```

Comment:
Vì sao sử dụng let để đặt biến mà không sử dụng var?

Reply:
Vì var biến toàn cục, còn let một khối mã, dùng let không bận tâm biến có dùng trùng tên với biến khai báo trước đó ở phạm vi tồn tại khác.